packages/client/src/components/Component.svelte

```diff
@@ -192,6 +193,7 @@
   let pad = false
   $: pad = pad || (interactive && hasChildren && inDndPath)
   $: $dndIsDragging, (pad = false)
+  $: isDark = !$themeStore.theme?.includes("light")
```

Comment:
Since we can use the normal `context` to get the theme (like we moved over to doing in the signature field, we can remove this from the core Component.